hyperparameter_tuning/rapids_bring_your_own/code/workflows/MLWorkflowSingleCPU.py

```diff
@@ -53,7 +53,7 @@ def ingest_data(self):
         if "Parquet" in self.hpo_config.input_file_type:
             hpo_log.info("> parquet data ingestion")
 
-            assert isinstance(self.hpo_config.target_files, str)
+            # assert isinstance(self.hpo_config.target_files, str)
```

Comment:
Can this be uncommented? Does the assertion fail?

Reply:
Yes, the assertion is not needed anymore since the data is read differently now (see `HPOConfig.py`). I'll remove the comment.

Reply:
Looks good. Thanks for the contribution!